test/org/sagebionetworks/bridge/dynamodb/DynamoStudyDaoTest.java

```diff
@@ -52,7 +52,7 @@ public void before() {
     public void after() {
         for (String oneStudyId : studyIdsToDelete) {
             try {
-                Study study = studyDao.getStudy(oneStudyId);
+                Study study = studyDao.getStudyOnlyForDao(oneStudyId);
                 studyDao.deleteStudy(study);
```

Comment:
Can a user delete a deactivated study? They should be able to do this. I don't think they should need to activate the study before they can delete it, either. This may require changing the delete to only take a studyId.